.github/workflows/release_request.yml

```diff
@@ -0,0 +1,64 @@
+name: Release request
```

Comment:
Please [reuse](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow) the [dotnet workflow](https://github.com/Adyen/adyen-dotnet-api-library/blob/0d14bba85449d026a80e2cc71ef210a41d8efb5a/.github/workflows/release.yml) to avoid copy pasting this workflow everywhere.